### PR TITLE
ER-1466 Make error page CTA content consistent

### PR DESCRIPTION
**Pull request by sachpatel**

### CODE REVIEW

File: src/Employer/Employer.Web/Views/Error/Error.cshtml [hunk 1]
```diff
@@ -5,6 +5,6 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl">There's been an error</h1>
-        <p>Sorry, there seems to be an error with this page. Please refresh your browser or <a href="/">go back to the homepage</a>.</p>
+        <p>Sorry, there seems to be an error with this page. Please refresh your browser or <a asp-route="@RouteNames.Dashboard_Account_Home">return to homepage</a>.</p>
```
Comment (shomavg) [inline src/Employer/Employer.Web/Views/Error/Error.cshtml]: Should the return be Return with capital R?

File: src/Employer/Employer.Web/Views/Error/PageNotFound.cshtml [hunk 2]
```diff
@@ -7,14 +7,12 @@
         <header class="page-header group">
             <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">Page not found</h1>
         </header>
-
         <div class="article-container">
             <article role="article" class="group">
                 <div class="inner">
-                    <p>Please check that you entered the correct web address. Alternatively, <a href="/">return to the homepage</a></p>
+                    <p>Please check that you entered the correct web address. Alternatively, <a asp-route="@RouteNames.Dashboard_Account_Home">return to homepage</a>.</p>
```
Comment (shomavg) [inline src/Employer/Employer.Web/Views/Error/PageNotFound.cshtml]: Same as above.. Return?

File: src/Provider/Provider.Web/Views/Error/Error.cshtml [hunk 3]
```diff
@@ -5,6 +5,6 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl govuk-!-margin-bottom-5">There's been an error</h1>
-        <p class="govuk-body">Sorry, there seems to be an error with this page. Please refresh your browser or <a asp-route="@RouteNames.Dashboard_Account_Home" class="govuk-link">go back to the homepage</a>.</p>
+        <p class="govuk-body">Sorry, there seems to be an error with this page. Please refresh your browser or <a asp-route="@RouteNames.Dashboard_Account_Home" class="govuk-link">return to homepage</a>.</p>
```
Comment (shomavg) [inline src/Provider/Provider.Web/Views/Error/Error.cshtml]: Return?

File: src/Provider/Provider.Web/Configuration/Routing/RouteNames.cs [hunk 4]
```diff
@@ -2,7 +2,6 @@
 {
     public static class RouteNames
     {
-        public const string BlockedProvider_Get= "BlockedProvider_Get";
```
Comment (dev4valtech) [inline src/Provider/Provider.Web/Configuration/Routing/RouteNames.cs]: Why was this removed?

File: src/QA/QA.Web/Views/Error/Error.cshtml [hunk 5]
```diff
@@ -5,6 +5,6 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl">There's been an error</h1>
-        <p class="govuk-body">Sorry, there seems to be an error with this page. Please refresh your browser or <a href="/">go back to the homepage</a>.</p>
+        <p class="govuk-body">Sorry, there seems to be an error with this page. Please refresh your browser or <a href="/">return to homepage</a>.</p>
```
Comment (shomavg) [inline src/QA/QA.Web/Views/Error/Error.cshtml]: Return?

File: src/Provider/Provider.Web/Configuration/ViewNames.cs [hunk 6]
```diff
@@ -17,7 +17,6 @@ public class ViewNames
         public const string PageNotFound = "PageNotFound";
         public const string VacancyPreview = "VacancyPreview";
         public const string CloseVacancyView = "Close";
-        public const string ProviderBlocked = "ProviderBlocked";
```
Comment (dev4valtech) [inline src/Provider/Provider.Web/Configuration/ViewNames.cs]: Same here, why was this removed?

File: src/QA/QA.Web/Views/Error/PageNotFound.cshtml [hunk 7]
```diff
@@ -10,7 +10,7 @@
         <div class="article-container">
             <article role="article" class="group">
                 <div class="inner">
-                    <p class="govuk-body">Please check that you entered the correct web address. Alternatively, <a href="/">return to the homepage</a></p>
+                    <p class="govuk-body">Please check that you entered the correct web address. Alternatively, <a href="/">return to homepage</a></p>
```
Comment (shomavg) [inline src/QA/QA.Web/Views/Error/PageNotFound.cshtml]: Return?

File: src/Employer/Employer.Web/Views/Error/Error.cshtml (hunk 1 above)
Comment (sachpatel) [reply src/Employer/Employer.Web/Views/Error/Error.cshtml]: no because it is in the middle of a sentence.

File: src/Employer/Employer.Web/Views/Error/PageNotFound.cshtml (hunk 2 above)
Comment (sachpatel) [reply src/Employer/Employer.Web/Views/Error/PageNotFound.cshtml]: no because it is in the middle of a sentence.

File: src/Provider/Provider.Web/Views/Error/Error.cshtml (hunk 3 above)
Comment (sachpatel) [reply src/Provider/Provider.Web/Views/Error/Error.cshtml]: same

File: src/QA/QA.Web/Views/Error/Error.cshtml (hunk 5 above)
Comment (sachpatel) [reply src/QA/QA.Web/Views/Error/Error.cshtml]: same

File: src/QA/QA.Web/Views/Error/PageNotFound.cshtml (hunk 7 above)
Comment (sachpatel) [reply src/QA/QA.Web/Views/Error/PageNotFound.cshtml]: same

File: src/Provider/Provider.Web/Configuration/Routing/RouteNames.cs (hunk 4 above)
Comment (sachpatel) [reply src/Provider/Provider.Web/Configuration/Routing/RouteNames.cs]: Because this is some leftover from the provider whitelist functionality which has now been removed but this stuff was remaining even though it was not used.

File: src/Provider/Provider.Web/Configuration/ViewNames.cs (hunk 6 above)
Comment (sachpatel) [reply src/Provider/Provider.Web/Configuration/ViewNames.cs]: mentioned above.
